### ISSUE TITLE
proxyOptions are never used in any request function.

### ISSUE THREAD

**Issue by holgor** [jgoralcz/aki-api#121]
I'm a Chinese Developer. https://cn.akinator.com/ can not be access at local environment without a VPN. I used the proxy options, but it didn't work as I thought. Finally, I checked your codes in master branch and found that the httpsAgent are never used in any request function.

**Comment by jgoralcz** [comment]
Please let me know if 6.0.8 fixes your problem

**Comment by holgor** [comment]
> Please let me know if 6.0.8 fixes your problem

Not quite. Maybe getServer and getSesstion are still not using the config. It might be better to call Axios.create to create a singleton instance for all common request instead of use the static functions of Axios.

**Comment by jgoralcz** [comment]
Good idea. I'll work on that later or you can make a PR

**Comment by jgoralcz** [comment]
Okay... over a year later, the updated code should be working 😓
